runtime/tools/python/ttrt/common/golden.py

```diff
@@ -148,11 +148,61 @@ def golden(binary, programContext, opContext):
             op_output_tensor, dtype=torch.float32
         ).flatten()
 
+        torch.save(golden_tensor_torch, f"{runtimeConfig.artifact_dir}/{loc}_golden.pt")
+        torch.save(output_tensor_torch, f"{runtimeConfig.artifact_dir}/{loc}_device.pt")
```

Comment:
I'm not sure this is desirable, for large models this will result in many gigabytes of intermediate data.  Maybe we can dump the tensors on mismatch? And probably have it under a ttrt flag disabled by default.

Reply:
fixed.